### DISCUSSION TITLE
Ultra Novice: after index.html build with npm, what goes to cpanel or external webhost?

### DISCUSSION THREAD

**Discussion by haufe7809n** [uswds/uswds#5251] [Q&A] [answered]
I didn't exactly know how to search this question. 

I have built a website using the MIT open source java & css [Edit: bootstrap, et al. via basic website folder tree], then copying that onto a commercial hosting provider. It worked! There were just a few folders: img, js, css... and my html pages--very easy.

I launched npm succesfully and followed the USWDS Tutorial. I am able to edit index.html to see features/edits I like. When I close terminal and npm stops running, index.html looks like a text-only html.

What would go onto my hosting provider? Would I include everything in my working folder, e.g. ~/uswds-tutorial 

If this question is redundant or time consuming to answer, I am happy to read recommended resources via links or tutorials.

EDIT:

Arranging the main uswds.css and javascript file, I was able to manually achieve what I was after. I'll take a look at NPM when more time is available.

**Comment by mejiaj** [comment, marked as answer]
@haufe7809n, locally NPM installs all required packages for you project and the `npm start` command starts a local server. Files like html, css, and javascript will require publishing to a server for others to see it. Github Pages offers a free method of hosting simple static (HTML, CSS, JS, but no database) websites.

If you'd like to learn more, MDN has a great guide on getting started with web development; [Publishing your website - Learn web development | MDN](https://developer.mozilla.org/en-US/docs/Learn/Getting_started_with_the_web/Publishing_your_website)

**Comment by ryparker** [comment]
You confirmed that publishing the CSS and JavaScript files resolved the hosting question, so I'm closing this as answered. The [accepted answer](https://github.com/uswds/uswds/discussions/5251#discussioncomment-5927852) remains available for anyone setting this up.
